sacn/messages/data_packet.py

```diff
@@ -170,23 +183,27 @@ def make_data_packet(raw_data) -> 'DataPacket':
            tuple(raw_data[40:44]) != tuple(VECTOR_E131_DATA_PACKET) or \
            raw_data[117] != VECTOR_DMP_SET_PROPERTY:  # REMEMBER: when slicing: [inclusive:exclusive]
             raise TypeError('Some of the vectors in the given raw data are not compatible to the E131 Standard!')
-        if raw_data[125] != 0x00:
-            raise TypeError(f'{raw_data[125]} is not a default Null Start Code for Dimmers per DMX512 & DMX512/1990')
 
-        tmpPacket = DataPacket(cid=raw_data[22:38], sourceName=str(raw_data[44:108]),
-                               universe=(0xFF * raw_data[113]) + raw_data[114])  # high byte first
+        tmpPacket = DataPacket(cid=raw_data[22:38], sourceName=bytes(raw_data[44:108]).decode('utf-8').replace("\0", ''),
```

Comment:
Small nitpick: please use `'` intstead of `"`, to stay consistent.
So that `"\0"` → `'\0'`